scripts/release.sh

```diff
@@ -18,6 +18,10 @@ if [[ ! $1 =~ ^[0-9]*\.[0-9]*\.[0-9]*(-[A-Z0-9]*)?$ ]]; then
     exit 1
 fi
 
+if [ -z $ANDROID_SDK_ROOT ]; then
+    echo "Android package will NOT be released"
```

Comment:
Shouldn't we abort rather abort the release as you want to normally release it together?

Reply:
yeah, that might be better